testing/test_xfail_behavior.py

```diff
@@ -121,9 +115,10 @@ def test_xfail(is_crashing, is_strict, testdir):
         @pytest.mark.forked
         def test_function():
             {test_func_body!s}
-        """.
-        format(**locals())
+        """.format(
```

Comment:
f string?

Reply:
Fixed. I also added `pyupgrade` to the pre-commit config.